dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/snowflake/SnowflakeLogsConnector.java

```diff
@@ -163,11 +163,6 @@ public void addTasksTo(List<? super Task<?>> out, ConnectorArguments arguments)
 
         String queryFormat = newQueryFormat(arguments);
 
-        final int daysToExport = arguments.getQueryLogDays(7);
-        if (daysToExport <= 0)
-            throw new MetadataDumperUsageException("At least one day of query logs should be exported; you specified: " + daysToExport);
-        LOG.info("Log entries within the last {} days will be exported in increments of 1 hour.", daysToExport);
```

Comment:
Why is this code being removed?

Reply:
This check is already being done inside the function at github.com/google/dwh-migration-tools/pull/133/files/d6d1173a85e4295443285c74c74e6fd6da865dc0#diff-7dcc102b5538a3c78cee78295c7c87e5ed1698fe369894cc6402e9785f85aaf4L175